torch/quantization/observer.py

```diff
@@ -700,9 +704,11 @@ def _non_linear_param_search(self):
         new_max = self.min_val + bin_width * (end_bin + 1)
         return new_min, new_max
 
+    @torch.jit.ignore
     def _combine_histograms(
         self, dst_histogram, dst_min, dst_max, src_histogram, src_min, src_max
     ):
+        # type: (Tensor, float, float, Tensor, float, float) -> Tensor
```

Comment:
Do you need this if the function is ignored?

Reply:
yeah, we need this otherwise everything will be treated as Tensor